downstream/modules/automation-hub/proc-shared-filesystem.adoc

```diff
@@ -0,0 +1,23 @@
+[id="proc-shared-filesystem"]
+
+= Setting up the shared filesystem
+
```

Comment:
Needs a procedure heading and a short description between this heading and the section title.